### PR TITLE
Allow users to disable APIs

### PR DESCRIPTION
**Pull request by vermakhushboo**
## What does this PR do?

Allow users to disable APIs 

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/controllers/api/projects.php [hunk 1]
```diff
@@ -473,6 +473,71 @@
...
+
+        $apis = $project->getAttribute('apis', []);
+        $apis[$api] = $status;
+
+        $project = $dbForConsole->updateDocument('projects', $project->getId(), $project->setAttribute('apis', $apis));
+
+        $response->dynamic($project, Response::MODEL_PROJECT);
+    });
+
+App::patch('/v1/projects/:projectId/api/all')
```
Comment (eldadfux) [inline app/controllers/api/projects.php:507]: Not sure I love this all endpoint, I see it was already there for services. Do we use it in the console? Can we remove both? Can also be addressed later.

File: app/controllers/general.php [hunk 2]
```diff
@@ -92,6 +92,13 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
         }
     }
 
+    if (array_key_exists('proxy', $project->getAttribute('apis', []))) {
```
Comment (eldadfux) [inline app/controllers/general.php:95]: What is proxy? is this for custom domains/functions?

File: app/controllers/api/projects.php (hunk 1 above)
Comment (vermakhushboo) [reply app/controllers/api/projects.php:507]: Yes, we use it in the console. As discussed, we can address it later.

File: app/controllers/general.php (hunk 2 above)
Comment (vermakhushboo) [reply app/controllers/general.php:95]: Probably used for functions. Just copied it from the existing services code.

**Pull request merged**